src/plugins/kibana_usage_collection/server/collectors/csp/csp_collector.test.ts

```diff
@@ -22,7 +22,13 @@ describe('csp collector', () => {
   const mockedFetchContext = createCollectorFetchContextMock();
 
   function updateCsp(config: Partial<ICspConfig>) {
-    httpMock.csp = new CspConfig(config);
+    httpMock.csp = new CspConfig({
+      ...CspConfig.DEFAULT,
+      style_src: [],
+      worker_src: [],
+      script_src: [],
+      ...config,
+    });
```

Comment:
Not really proud of that, but the alternative was to publicly expose the `CspConfigType` type from `core` just for that, and it felt worse.